pkg/manager/impl/validation/validation.go

```diff
@@ -268,3 +276,21 @@ func ValidateOutputData(outputData *core.LiteralMap, maxSizeInBytes int64) error
 	}
 	return errors.NewFlyteAdminErrorf(codes.ResourceExhausted, "Output data size exceeds platform configured threshold (%+v > %v)", outputSizeInBytes, maxSizeInBytes)
 }
+
+func ValidateDatetime(literal *core.Literal) error {
+	if literal == nil {
+		return nil
+	}
+
+	timestamp := literal.Value.(*core.Literal_Scalar).Scalar.Value.(*core.Scalar_Primitive).Primitive.GetValue().(*core.Primitive_Datetime).Datetime
```

Comment:
```suggestion
	timestamp := literal.GetScalar().GetPrimitive().GetDatetime()
```

Reply:
(also because of the DCO requirement please don't actually accept this suggestion from the Github UI because it won't be signed off 😅 )

Reply:
Thanks a lot for the suggestion! Now it's done :)